docs/source/explanation/explanation_of_numerical_optimizers.md

```diff
@@ -14,8 +14,9 @@ The main principles we describe here are:
 - Derivative free trust region algorithms
 - Derivative free direct search algorithms
 
-This covers a large range of the algorithms that come with optimagic. We do currently
-not cover:
+This covers a large range of the algorithms that come with optimagic. In contrast, the
```

Comment:
Not sure whether I got this right, but the current version is unclear about where these optimizers are not covered (in this overview or in optimagic in general)